_data/python-pfhub/README.md

```diff
@@ -0,0 +1,22 @@
+# Nix
```

Comment:
Please include a one-line summary of what python-pfhub is meant to accomplish, at least, insert a top-level `# PFHub Python Package`, and increase the existing headings by a level (e.g., `# Nix` -> `## Nix`).

Reply:
Included short description